query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/ServicesQueries.java

```diff
@@ -53,6 +53,56 @@ static QueryRequest buildQuery1() {
             .build());
 
 
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceId));
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceName));
+    Function serviceDurationPercentileFunc = Function.newBuilder().addArguments(
+        Expression.newBuilder().setColumnIdentifier(ColumnIdentifier.newBuilder().setColumnName("SERVICE.duration").build()).build()).setFunctionName("PERCENTILE99")
+        .build();
+    OrderByExpression orderByExpression = OrderByExpression.newBuilder().setOrder(SortOrder.DESC).setExpression(
+        Expression.newBuilder().setFunction(serviceDurationPercentileFunc).build()).build();
+    builder.addOrderBy(orderByExpression);
+    builder.setLimit(10000);
+
+    return builder.build();
+  }
+
+  static QueryRequest buildQuery2() {
```

Comment:
Can we add test for `buildAvgRateQueryWithTimeAggregation`? This will test the time-series functionality with avg_rate.

Reply:
What needs to be tested in time series specifically? As I can see at query service level, both metric series and time series request seem similar.